Strategy11/formidable-forms
# Set aria-invalid to true only on name part input with error

Fix https://github.com/Strategy11/formidable-pro/issues/5409

### Test steps
1. Create a form and add a name field and set it to be required.
2. Preview your form and try submitting the form filling part of the name field at a time (either first, middle or last).
3. Confirm that the submission is failed and the first field with error is focused on.
4. Confirm that `aria-invalid` attribute is only set on fields with error.

### Screen record
![CleanShot 2024-10-09 at 15 48 22](https://github.com/user-attachments/assets/ca0abd51-867c-41f7-a2c6-6ffebf376a56)
